provider/gce/environ_network_test.go

```diff
@@ -238,11 +240,132 @@ func (s *environNetSuite) TestNetworkInterfaceInvalidCredentialError(c *gc.C) {
 	})
 	s.FakeEnviron.Insts = []instances.Instance{s.NewInstanceFromBase(baseInst)}
 
-	_, err := s.NetEnv.NetworkInterfaces(s.CallCtx, instance.Id("moana"))
+	_, err := s.NetEnv.NetworkInterfaces(s.CallCtx, []instance.Id{instance.Id("moana")})
 	c.Check(err, gc.NotNil)
 	c.Assert(s.InvalidatedCredentials, jc.IsTrue)
 }
 
+func (s *environNetSuite) TestInterfacesForMultipleInstances(c *gc.C) {
+	s.cannedData()
+	baseInst1 := s.NewBaseInstance(c, "i-1")
+
+	// Create a second instance and patch its interface list
+	baseInst2 := s.NewBaseInstance(c, "i-2")
+	b2summary := &baseInst2.InstanceSummary
+	b2summary.NetworkInterfaces = []*compute.NetworkInterface{
+		{
+			Name:       "netif-0",
+			NetworkIP:  "10.0.10.42",
+			Network:    "https://www.googleapis.com/compute/v1/projects/sonic-youth/global/networks/go-team",
```

Comment:
Am I right in thinking that this is sample data similar to what the gce api gives us?

Reply:
This was copied from another test in the package and adapted for this particular test. I assume that the format matches the gce API output.